runtime-testsuite/test/org/antlr/v4/test/runtime/cpp/BaseCppTest.java

```diff
@@ -214,7 +214,7 @@ private boolean buildRuntime() {
 		System.out.println("Building ANTLR4 C++ runtime (if necessary) at "+ runtimePath);
 
 		try {
-			String[] command = { "cmake", ".", /*"-DCMAKE_CXX_COMPILER=clang++",*/ "-DCMAKE_BUILD_TYPE=release" };
+			String[] command = { "cmake", ".", /*"-DCMAKE_CXX_COMPILER=clang++",*/ "-DCMAKE_BUILD_TYPE=debug" };
```

Comment:
Could it affect performance? We've changed the configuration to Release for C#, Swift for performance improvement.

Reply:
Ugh, I mean to revert this. I had to switch between Mac and Linux to hunt down a bug. I'll get rid of this.

Reply:
Fixed.